plugins/explore/src/components/GroupsExplorerContent/GroupsDiagram.tsx

```diff
@@ -91,19 +109,86 @@ function RenderNode(props: DependencyGraphTypes.RenderNodeProps<any>) {
         })}
       >
         <text
-          x={90}
-          y={45}
+          x={NODE_WIDTH / 2}
+          y={NODE_HEIGHT / 2 + NODE_MIDDLE_ALIGNMENT_SHIFT}
           textAnchor="middle"
           alignmentBaseline="baseline"
-          style={{ fontWeight: 'bold' }}
+          style={{ fontWeight: 'bold', fontSize: TEXT_FONT_SIZE }}
         >
-          {props.node.name}
+          {objs.map(function (object: any, i: number) {
+            return (
+              <tspan
+                key={i}
+                y={NODE_HEIGHT / 2 + NODE_MIDDLE_ALIGNMENT_SHIFT}
+                x="90"
+                textAnchor="middle"
+                dy={object.dy}
+              >
+                {object.text}
+              </tspan>
+            );
+          })}
         </text>
       </Link>
     </g>
   );
 }
 
+/**
+ * Join chunks based on max words per line and add dy position shifting
+ * @param chunkedArray
+ * @returns
+ */
+function prepareForDisplay(chunkedArray: any): any {
```

Comment:
let's avoid `any`. Just `Array<{ dy: number, text: string }>` works here

Reply:
removed this part from the code